sdk/formrecognizer/perf-tests/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@azure-tests/perf-ai-form-recognizer",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Performance Tests for @azure/ai-form-recognizer",
+  "main": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
NIT: use "dependencies" for stuff imported by the tests and "devDependencies" for stuff needed to run them.

Reply:
Hmm, storage perf tests probably have this problem, too, since I copied from there.